Doc/library/sqlite3.rst

```diff
@@ -222,14 +222,16 @@ Module functions
 
 .. function:: complete_statement(statement)
 
-   Returns ``True`` if the string *statement* contains one or more complete SQL
-   statements terminated by semicolons. It does not verify that the SQL is
-   syntactically correct, only that there are no unclosed string literals and the
-   statement is terminated by a semicolon.
+   Return ``True`` if the string *statement* appears to contain one or more
+   complete SQL statements terminated by semicolons.
+   No syntactic verification or parsing of any kind is performed.
```

Comment:
Reading the new version (before reading the old), it was very confusing to me what it _does_ do, if it doesn't do any syntactic verification or parsing of any kind, whereas reading the old version it was then much more clear. I suggest either reverting to something more similar to the latter, or rewriting this in a different, more explicit way.

One possibility combining the two:

```suggestion
   Return ``True`` if the string *statement* appears to contain
   one or more complete SQL statements.
   No syntactic verification or parsing of any kind is performed,
   other than checking that there are no unclosed string literals
   and the statement is terminated by a semicolon.
```

Reply:
True, that's better.